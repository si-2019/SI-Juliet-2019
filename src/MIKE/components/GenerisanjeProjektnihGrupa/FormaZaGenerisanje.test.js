import React from 'react';
import GenerisanjeProjektneGrupe from './FormaZaGenerisanje'
import Enzyme, { configure, shallow, mount, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
 

Enzyme.configure({ adapter: new Adapter() });

it("Da li se prikazuju komponente", ()=> {
    const component=shallow(<GenerisanjeProjektneGrupe />);
    expect(component).toMatchSnapshot();
});

it("Da li postoji forma za generisanje projektne grupe", ()=> {
    const wrapper=shallow(<GenerisanjeProjektneGrupe />);
    expect(wrapper.find("#formaGenerisanje")).toBeDefined();
});

it("Da li postoji select koji prikazuje predmete za koje je moguce kreirati projektnu grupu", ()=> {
    const wrapper=shallow(<GenerisanjeProjektneGrupe />);
    expect(wrapper.find("#selectPredmeti")).toBeDefined();
 
});

it("Da li je omogucen odabir broja studenata po grupi", ()=> {
    const wrapper=shallow(<GenerisanjeProjektneGrupe />);
    expect(wrapper.find("#brojStudenata")).toBeDefined();
 
});

it("Da li je omogucen odabir redoslijeda generisana studenata po grupi", ()=> {
    const wrapper=shallow(<GenerisanjeProjektneGrupe />);
    expect(wrapper.find("#checkbox")).toBeDefined();
 
});

