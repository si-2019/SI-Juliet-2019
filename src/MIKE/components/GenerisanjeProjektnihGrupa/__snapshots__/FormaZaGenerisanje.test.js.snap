// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Da li se prikazuju komponente 1`] = `ShallowWrapper {}`;
